.github/workflows/dotnet.yml

```diff
@@ -76,7 +70,6 @@ jobs:
     name: "Publish NuGet package"
     runs-on: ubuntu-latest
     needs: build-and-test
-    if: github.event_name == 'push'
```

Comment:
PLEASE don't publish on every successful PR build - this leads to a lot of manual work to later unlist all the packages of the PR that have a higher version number than our regular pre-release packages.